joomla/joomla-cms
# com_installer: translatable ordering in manage and updatesites views

#### Description

This PR intents to solve the issue in two views of com_installer where ordering of some columns in some views (manage and updatesites) are not correct when using other languages that not english.
#### How to test
1. Use latest staging
2. Install, for instance, german language and set it as default admin
3. Go to Erweiterungen -> Verwalten -> Manage (Extensions -> Manage -> Manage in english)
4. Now order by "Typ" (Type), you will see that is not ordered correctly.
5. Install this patch.
6. Repeat steps 3 and 4. You will see is ordered correctly now.
7. Do the similar process for Erweiterungen -> Verwalten -> Update Sites (Extensions -> Manage -> Update Sites in english)
8. Test also the search, other ordering, pagination and filters in those two views and check everything is fine.

It would be nice to check other languages to. I tried also pt-PT and works fine.

@richard67 here is the last PR for the translate ordering problem in com_install.
